compiler/rustc_query_system/src/query/plumbing.rs

```diff
@@ -11,7 +11,6 @@ use crate::query::job::QueryLatch;
 use crate::query::job::{report_cycle, QueryInfo, QueryJob, QueryJobId, QueryJobInfo};
 use crate::query::SerializedDepNodeIndex;
 use crate::query::{QueryContext, QueryMap, QuerySideEffects, QueryStackFrame};
-use crate::values::Value;
```

Comment:
The trait and its can now be grouped in `rustc_middle::query::values`.

Reply:
Maybe add an `impl<T, C> !Value<C> for Erased<T>` to avoid jokes?

Reply:
I ran into specialization errors trying to remove the `unsafe` `Value` impls in `rustc_middle`, so I think I'll just leave it in `rustc_query_system`.